src/nni_manager/training_service/kubernetes/kubernetesTrainingService.ts

```diff
@@ -81,14 +81,14 @@ abstract class KubernetesTrainingService {
         }
     }
 
-    public listTrialJobs(): Promise<TrialJobDetail[]> {
+    public async listTrialJobs(): Promise<TrialJobDetail[]> {
         const jobs: TrialJobDetail[] = [];
         
-        this.trialJobsMap.forEach(async (value: KubernetesTrialJobDetail, key: string) => {
+        for (const [key, value] of this.trialJobsMap) { 
```

Comment:
key, value without type definition, could it pass tslint ?

Reply:
destructuring assignment does not support declare type.